src/azure-cli/azure/cli/command_modules/aro/tests/latest/test_aro_commands.py

```diff
@@ -45,13 +46,14 @@ def test_aro_create(self, resource_group):
         self.cmd('network vnet subnet create -g {rg} --vnet-name dev-vnet -n {worker_subnet} --address-prefixes {worker_ip_range} --service-endpoints Microsoft.ContainerRegistry')
         self.cmd('network vnet subnet update -g {rg} --vnet-name dev-vnet -n {master_subnet} --disable-private-link-service-network-policies true')
 
-        self.cmd('aro create -g {rg} -n {name} --master-subnet {master_subnet_resource} --worker-subnet {worker_subnet_resource} --subscription {subscription} --tags test=create', checks=[
-            self.check('tags.test', 'create'),
-            self.check('name', '{name}'),
-            self.check('masterProfile.subnetId', '{master_subnet_resource}'),
-            self.check('workerProfiles[0].subnetId', '{worker_subnet_resource}'),
-            self.check('provisioningState', 'Succeeded')
-        ])
+        with mock.patch('azure.cli.command_modules.aro._rbac._gen_uuid', side_effect=self.create_guid):
+            self.cmd('aro create -g {rg} -n {name} --master-subnet {master_subnet_resource} --worker-subnet {worker_subnet_resource} --subscription {subscription} --tags test=create', checks=[
+                self.check('tags.test', 'create'),
+                self.check('name', '{name}'),
+                self.check('masterProfile.subnetId', '{master_subnet_resource}'),
+                self.check('workerProfiles[0].subnetId', '{worker_subnet_resource}'),
+                self.check('provisioningState', 'Succeeded')
+            ])
 
     @AllowLargeResponse()
```

Comment:
I know, I had to remove this line to make it work in my PR. https://github.com/Azure/azure-cli/pull/13086/files#diff-36af8ad76e1e927f0a66611dfd361a53L56

Reply:
I believe it will help here

Reply:
Added this as well.  Let's see what happens.